Dockerfile

```diff
@@ -0,0 +1,33 @@
+# BUILD IMAGE --------------------------------------------------------
+FROM nimlang/nim:1.2.0-alpine AS nim-build
+
+# Get build tools and required header files
+RUN apk add --no-cache bash build-base pcre-dev linux-headers
+
+RUN mkdir /app
+WORKDIR /app
+ADD . .
+
+# Ran separately from 'make' to avoid re-doing
+RUN git submodule update --init --recursive
+
+# Build the actual node binary
+RUN make wakunode
+
+# ACTUAL IMAGE -------------------------------------------------------
+FROM alpine:3.11
+
+LABEL maintainer="jakub@status.im"
+LABEL source="https://github.com/status-im/nim-waku"
+LABEL description="nim-waku is a Waku protocol node written in Nim"
```

Comment:
Perhaps use the original description I had? As it is really a node that talks waku but e.g. also still Whisper.

Reply:
Sure.